qwalk_worker.py

```diff
@@ -455,7 +477,10 @@ def list_dir(d: ListDirArgs, ww: "QWalkWorker") -> Sequence[str]:
                     time.sleep(5)
                     log_it("HTTP API error: %s" % re.sub(r"[\r\n]+", " ", str(e))[:100])
                     log_it("id: %s - next_uri: %s" % (d["path_id"], next_uri))
-                    ww.rc.login(ww.creds["QUSER"], ww.creds["QPASS"])
+                    if ww.creds["QTOKEN"]:
+                        ww.rc = RestClient(random.choice(ww.ips), ww.ww.creds.get("QPORT", REST_PORT), Credentials(ww.creds["QTOKEN"]))
+                    else:
+                        ww.rc.login(ww.creds["QUSER"], ww.creds["QPASS"])
```

Comment:
This might be solved when moving to a function, but let's make sure that none of the lines are longer than 100 characters